### PR TITLE
Login errors

### PR DESCRIPTION
**Pull request by adriana-alter**
screen shot 2019-01-30 at 12 41 29 pm

### CODE REVIEW

File: src/stylesheets/reusables/form.scss [hunk 1]
```diff
@@ -22,6 +22,12 @@ form {
     }
   }
 
+  .alert.alert-danger {
```
Comment (EdwinGuzman) [inline src/stylesheets/reusables/form.scss]: Can you make this a child instead?

**Comment by EdwinGuzman** [comment]
Looks good. Just make it focus on the error message when it appears and you're all set.

File: src/stylesheets/reusables/form.scss (hunk 1 above)
Comment (adriana-alter) [reply src/stylesheets/reusables/form.scss]: What do you mean--make it a child of what?

Comment (EdwinGuzman) [reply src/stylesheets/reusables/form.scss]:
````
  .alert {
    &.alert-danger { 
     .....
```

**Pull request merged**